src/theme/variables.scss

```diff
@@ -41,6 +41,13 @@ $colors: (
   light:      #f4f4f4,
   dark:       #222
 );
+// $alert-min-width: (	
```

Comment:
If you test out styles and they don't work, it's best not to pollute a global file with it. This is relatively small change, so we can take it out but just keep that in mind for the future.